app/presenters/curation_concerns/generic_work_show_presenter.rb

```diff
@@ -78,5 +78,11 @@ def show_thumb_member_presenters
         end
       end
     end
+
+    def riiif_file_id
+      return unless member_presenters.present?
+      representative_presenter = member_presenters.select{ |pres| pres.id == representative_id }.pop
+      representative_presenter.riiif_file_id
+    end
```

Comment:
looks like there's already a way to grab this, which I will use! https://github.com/samvera/curation_concerns/blob/master/app/presenters/curation_concerns/work_show_presenter.rb#L47